themes/powerline-multiline/powerline-multiline.theme.bash

```diff
@@ -153,7 +153,7 @@ function __powerline_battery_prompt {
     else
       color="${BATTERY_STATUS_THEME_PROMPT_GOOD_COLOR}"
     fi
-    [[ "$(ac_adapter_connected)" ]] && battery_status="${BATTERY_AC_CHAR}${battery_status}"
+    ac_adapter_connected && battery_status="${BATTERY_AC_CHAR} ${battery_status}"
```

Comment:
The space between `{BATTERY_AC_CHAR}` and `${battery_status}"` is not necessary, I personally prefer the "ac symbol" alone, without this space, if someone wants the space, can modify the `BATTERY_AC_CHAR` and add it.

What do you think?